cnd/src/storage.rs

```diff
@@ -39,6 +41,40 @@ impl Storage {
             halight_states,
         }
     }
+
+    pub async fn save_what_alice_learned_from_bob(
```

Comment:
What do you need these for if you have the trait impl?

Reply:
I wanted to add helper functions, its either add them here as `Storage` methods or pass in an additional parameter `storage`, methods is nicer.

Reply:
Turns out these are no good anyways once we add the other trading pairs. Removed.